### ISSUE TITLE
Fusion of the caudate and putamen

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7317]
**Preferred term label:**
Fusion of the caudate and putamen 

e. The basal ganglia appear as large round structures with (arrowheads) on axial T2-weighted image. 
**Synonyms**
Fused caudate and putamen 

**Definition (free text, please give PubMed ID)**
Abnormal joined appearance of the caudate and putamen with an absence of the anterior limb of the internal capsule
PMID:31194327
PMID:26251641

Comment:
The putamen, combined with the globus pallidus, forms the lentiform nucleus; and with the caudate nucleus, it shapes the striatum, which is a subcortical structure that forms the basal ganglia.

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM:614039 
8/9  PMID:20829227

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
